package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "pgmock2",
-  "version": "2.0.5",
+  "version": "2.0.8",
```

Comment:
We'll need to change this to `2.0.6` to match up with current release (i.e. `2.0.5`).